examples/dart/test/data_types_test.dart

```diff
@@ -62,4 +71,40 @@ main() {
     // :snippet-end:
     expect(object.id.toString(), isA<String>());
   });
+  test('DateTime', () {
+    // :snippet-start: datetime-use
+    final config = Configuration.local([Vehicle.schema]);
+    final realm = Realm(config);
+
+    // Create a Realm object with date in UTC, or convert with .toUtc() before storing
+    Vehicle subaruOutback = realm.write<Vehicle>(() {
+      return realm.add(Vehicle('Subie', DateTime.utc(2022, 9, 18, 12, 30, 0)));
+    });
+    // :remove-start:
+    expect(
+        subaruOutback.dateLastServiced.toString(), '2022-09-18 12:30:00.000Z');
+    // :remove-end:
+
+    final fordFusion = Vehicle('Fuse', DateTime(2022, 9, 18, 8, 30, 0).toUtc());
+    realm.write(() {
+      realm.add(fordFusion);
+    });
+
+    // When you query the object, the `DateTime` returned is UTC
+    final queriedSubaruOutback =
+        realm.all<Vehicle>().query('nickname == "Subie"').first;
```

Comment:
```suggestion
        realm.all<Vehicle>().query('nickname == "Subie"')[0];
```

more common to access index, i believe. this is what we do in the rest of the code examples anyways